### DISCUSSION TITLE
How to set all inputs, select and items a default height?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#41049] [Q&A]
Hi guys, how i can set a default height for a better customization of antd?

**Comment by Wxh16144** [comment]
You can set all component sizes with `< ConfigProvider componentSize="small" />`. Documentation: https://ant.design/components/config-provider#components-config-provider-demo-size

**Reply by vitalijalbu** [reply]
> You can set all component sizes with `< ConfigProvider componentSize="small" />`. Documentation: https://ant.design/components/config-provider#components-config-provider-demo-size

It's possible to set size generally to all components??

**Reply by justirva09** [reply]
if you want set all globally and directly with your own preset, you can use like this:

```
<ConfigProvider
      theme={{
        token: {
          controlHeightSM: 35, // Small-sized controls height
          controlHeightLG: 35, // Large-sized controls height
          controlHeight: 35,   // Default control height
        },
      }}
    >
    ...
 </ConfigProvider>
```

cc: @vitalijalbu

**Comment by monolithed** [comment]
In the input, there are `paddingBlock` and `paddingInline`, while the select only has `paddingSM`. Universally, only `controlHeight` works.
